zmon-controller-app/pom.xml

```diff
@@ -225,6 +225,10 @@
             <version>0.4.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jooq</groupId>
```

Comment:
I have no idea. Sometimes Eclipse wants to be to clever and adds some entries in pom.xml because of wrong imports in java sources. But when you fix the java sources import, the pom.xml is not fixed. I will remove that.